spring-cloud-skipper-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -137,6 +137,286 @@ include::{snippets}/about-documentation/get-meta-information/http-response.adoc[
 
 include::{snippets}/about-documentation/get-meta-information/response-fields.adoc[]
 
+[[resources-upload]]
+=== Upload
+
+==== Upload a package
+
+TBD
```

Comment:
@ghillert I still see some TBD entries. Are these intended to be addressed in a separate PR or are you planning to update this PR.